src/libraries/Common/tests/System/Security/Cryptography/MLKemContractTests.cs

```diff
@@ -1057,7 +1057,6 @@ private static void AssertSameBuffer(ReadOnlySpan<byte> buffer1, ReadOnlySpan<by
             {
                 Assert.Fail("Expected buffers to be the same memory location, but were not.");
             }
-
```

Comment:
The file history on MLKemContractTests has no reason to say that it was edited by this change.  Please undo this edit, and the edit to MLKemBaseTests.